mongodbatlas/resource_mongodbatlas_database_user.go

```diff
@@ -105,6 +105,25 @@ func resourceMongoDBAtlasDatabaseUser() *schema.Resource {
 					},
 				},
 			},
+			"scopes": {
+				Type:     schema.TypeList,
```

Comment:
Just a question for my own curiosity, but labels is the same format in the API so why is that a TypeSet and this a TypeList?  I don't see why ordered vs unordered would be different?

Reply:
You're right, I'll change it, is better same like labels with TypeSet, thank you so much